src/main/resources/templates/misc/auto-rename.html

```diff
@@ -20,8 +20,29 @@
               <form method="post" enctype="multipart/form-data" th:action="@{'/api/v1/misc/auto-rename'}">
                 <div th:replace="~{fragments/common :: fileSelector(name='fileInput', multiple=false, accept='application/pdf')}"></div>
                 <br>
+                <div class="form-group">
+                  <label for="keyword">Keyword:</label>
+                  <input type="text" class="form-control" id="keyword" name="keyword" placeholder="e.g., Company, Name, Invoice" required>
```

Comment:
What about if people want to auto grab the title without using any keywords like previous functionality, we still need to support that and make it clear to use this is option, 
Perhaps a param to change from 
Autodetect from title vs using keyword
and have options show based on that